jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java

```diff
@@ -95,6 +96,25 @@ public void testEntrypoint_defaultWebAppRoot() throws IOException {
     Assert.assertEquals("ENTRYPOINT [\"java\",\"-jar\",\"/jetty/start.jar\"]", getEntrypoint());
   }
 
+  @Test
+  public void testUser() throws IOException {
+    Mockito.when(containerParameters.getUser()).thenReturn("tomcat");
+    task.generateDockerContext();
+
+    Assert.assertEquals("USER tomcat", getUser());
+  }
+
+  @Test
+  public void testUser_null() throws IOException {
+    Mockito.when(containerParameters.getUser()).thenReturn(null);
+    task.generateDockerContext();
+    try {
+      getUser();
+      Assert.fail();
+    } catch (NoSuchElementException ex) {
+    }
```

Comment:
nit: just add a `// pass` comment to the empty block here

Reply:
done